src/definitions/lightspeed.ts

```diff
@@ -61,6 +61,8 @@ export const LIGHTSPEED_SERVICE_LOGIN_TIMEOUT = 120000;
 
 export type LIGHTSPEED_SUGGESTION_TYPE = "SINGLE-TASK" | "MULTI-TASK";
 
+export const LIGHTSPEED_DEFAULT = "default";
```

Comment:
Minor comment, can we change `LIGHTSPEED_DEFAULT` to `LIGHTSPEED_MODEL_DEFAULT` to me more precise.